backend/app/categories/controller.py

```diff
@@ -17,9 +18,23 @@
 async def get_category_route(category_id: str,
                              db: AsyncIOMotorClient = Depends(get_database)) -> ItemsInResponse:
     """Get the items under a given category"""
-    # TODO add fetching from category
-    return ItemsInResponse()
-
+        
+    loader = None
+    if category_id == CategoryEnum.movies.value:
+        loader = Movie
+    if category_id == CategoryEnum.anime.value:
+        loader = Anime
+    if category_id == CategoryEnum.music.value:
+        loader = Music
+    if category_id == CategoryEnum.shows.value:
+        loader = Show
+    if category_id == CategoryEnum.books.value:
+        loader = Book
+    if category_id in CategoryEnum.__members__:
+        items = [loader(**item) async for item in db[category_id]["data"].find()]
+        return ItemsInResponse(data=items)
+    else:
+        return ItemsInResponse(success=False, error=[f"Invalid category {category_id}"])
```

Comment:
Returning ```ResponseBase(success=False, error=[f"Invalid category {category_id}"])``` would return an error as the return type had been declared to ItemsInResponse and ResponseBase is a parent class to it.